connect_four/connect_four.rb

```diff
@@ -1 +1,73 @@
-# Your code here!
\ No newline at end of file
+require 'pry'
+require 'pry-byebug'
+require './board.rb'
+require './player.rb'
+require './computer.rb'
+
+
+class Game
+
+  def initialize
+    @player1 = Player.new("X")
+    @player2 = Player.new("O")
+    @current_player = @player1
+    @game_board = Board.new
+  end
+
+  # The main play loop
+  def play
+    welcome
+    loop do
+      #lines 17-19 are my favorite
+      until @game_board.drop_token(@current_player.get_move_column,@current_player.game_token) do 
+        puts "Column full! Please choose another column!"
+        binding.pry
+        print ">"
+      end
+      @game_board.board_render
+      swap_players
+    end
+  end
+
+  def swap_players
+      if @current_player == @player1
+        @current_player = @player2
+      else
+        @current_player = @player1
+      end
+  end
+
+  def welcome
+    puts "Welcome to Connect Four. Play against a computer or human? "
+    game_answer = gets.strip.chomp
+    if game_answer == "human"
+    elsif game_answer == "computer"
+      @player2 = Computer.new("O")
+    end
```

Comment:
Use a better variable name here, like "number_of_players" or "who_is_player_two". This could be simplified into a single `if game_answer == "computer"` statement. You also might want to validate this input. As a game architecture level comment: You should get this input before assigning Player O to a human and then over-writing if it's going to be an AI.
`if number_of_players == 1
  @player2 = Player.new("O")
else
  @player2 = Computer.new("O")
end`